src/pages/fake-candidate-details/fake-candidate-details.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+import { BasePageComponent } from '../../shared/classes/base-page';
+
+import {
+  IonicPage,
+  NavController,
+  Platform,
+} from 'ionic-angular';
+
+import { AuthenticationProvider } from '../../providers/authentication/authentication';
+
+@IonicPage()
+@Component({
+  selector: 'page-fake-candidate-details',
```

Comment:
It's under the pages folder, so it's a page by default. Can we name the selector to `fake-candidate-details`?

Also you could initiate an html templet in this PR so that we have all of the boilerplate

Reply:
All pages have the prefix page in the app , changing it would break the pattern.